benches/benchmarks.rs

```diff
@@ -57,6 +59,140 @@ fn setup_engine() -> EngineState {
 // an executable for every single one - incredibly slowly. Would be nice to figure out
 // a way to split things up again.
 
+#[divan::bench]
+fn load_standard_lib(bencher: divan::Bencher) {
+    let engine = setup_engine();
+    bencher
+        .with_inputs(|| engine.clone())
+        .bench_values(|mut engine| {
+            load_standard_library(&mut engine).unwrap();
+        })
+}
+
+#[divan::bench_group]
+mod eval_commands {
+    use super::*;
+
+    #[divan::bench(args = [100, 1_000, 10_000])]
+    fn interleave(bencher: divan::Bencher, n: i32) {
+        let mut engine = setup_engine();
+        load_standard_library(&mut engine).unwrap();
```

Comment:
Do we always want to load the standard library? It may make sense to stress-test `EngineState` behavior when it already contains a certain amount of code but, shouldn't matter as much for simply testing the processing of data in commands and pipelines it probably doesn't matter as much.

Reply:
I don't think its needed, but i left it in as i think its a little more representative of how the state of the engine is when running it for the end user.

Reply:
As it is outside the timed section it shouldn't add too much noise if we change things there. So let's go with this for now. (we may want to have some benchmarks contrasting the different conditions just to gauge the impact of particular operations but step by step.